releasenotes/notes/os-capacity-deploy-hook-b52e87c0819df6fd.yaml

```diff
@@ -0,0 +1,17 @@
+---
+features:
+  - |
+    Automatic deployment for OpenStack Capacity via a Kayobe service
+    deploy hook using kolla admin credentials.
+issues:
+  - |
+    Will fail service deploy if correct authentication is not used for
+    os_capacity exporter and if kolla credentials are invalid or do not
+    have suitable permissions.
+upgrade:
+  - |
+    Remove application credentials used for OpenStack Capacity and replace
+    with OpenStack kolla credentails defined in admin-openrc.
```

Comment:
I'd make it a bit more clear that action is required by the user here. e.g.

```
    OpenStack Capacity no longer uses application credentials. Please delete any previously generated application credentials.
```

Reply:
Changed